src/machine/behavior/is/bitwise/sadd.rs

```diff
@@ -1,6 +1,18 @@
 use machine::state::State;
 
-pub fn sadd(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
+/// sideways add
+pub fn sadd(state: &mut State, x: u8, y: u8, z: u8) {
+    // calculate andn and load result
+    super::andn(state, x, y, z);
```

Comment:
Wouldn't that mean lots of redundant code?
Why not simply trace which function is called by the control unit? I'm not sure that I've understood correctly what you mean with "instruction tracing".